tests/test_relation_judge_634.py

```
@@ -565,10 +565,10 @@ def close_scene(_inputs):
             on_closing=closing_seen.set,
         )))
         worker.start()
-        assert closing_seen.wait(2) and judge_entered.wait(2) and scene_entered.wait(2)
+        closing_seen.wait(); judge_entered.wait(); scene_entered.wait()
```

Ruff (0.16.3)

[error] Multiple statements on one line (semicolon)

(E702)